packages/desktop/components/popup/PopupTemplate.svelte

```diff
@@ -91,7 +93,7 @@
                         disabled={busy || _backButton.disabled}
                         width="full"
                         form={_backButton.type === 'submit' ? _backButton.form : undefined}
-                        on:click={_backButton.type === 'button' && _backButton.onClick}
+                        on:click={_backButton.type === 'button' ? _backButton.onClick : undefined}
```

Comment:
Why did you change this?

Reply:
on:click expects an event handler function or undefined but this expression can return the first statement which is a boolean. Also, it was highlighting an error in my VSCode.